update-lit.sh

```diff
@@ -0,0 +1,155 @@
+#!/bin/bash
+
+cecho() {
+        # foreground \033[48;2;RRR;GGG;BBBm
+        # background \033[38;2;RRR;GGG;BBBm
+        # clear      \033[0m
+
+        if [ -z ${BASH_VERSION} ]; then # bash interpreter
+                echo "\033[38;2;50;210;255m$1\033[0m";
+        else # sh interpreter
+                echo -e "\033[38;2;50;210;255m$1\033[0m";
+        fi
+}
+
+while test $# -gt 0; do
+	case "$1" in
```

Comment:
These could all be handled by `LUVI_VERSION` and `LIT_VERSION` environment variables.